vsintegration/src/FSharp.Editor/Classification/ClassificationDefinitions.fs

```diff
@@ -73,13 +82,12 @@ module internal ClassificationDefinitions =
             let themeService = serviceProvider.GetService(typeof<SVsColorThemeService>) :?> IVsColorThemeService
             themeService.CurrentTheme.ThemeId
 
-        let colorData = // name,                  (light,                            dark)
-          [ FSharpClassificationTypes.Function,   (Colors.Black,                     Color.FromRgb(220uy, 220uy, 220uy))
-            FSharpClassificationTypes.MutableVar, (Color.FromRgb(160uy, 128uy, 0uy), Color.FromRgb(255uy, 210uy, 28uy))
-            FSharpClassificationTypes.Printf,     (Color.FromRgb(43uy, 145uy, 175uy), Color.FromRgb(78uy, 220uy, 176uy))
-            FSharpClassificationTypes.Property,   (Colors.Black,                     Color.FromRgb(220uy, 220uy, 220uy)) 
-            FSharpClassificationTypes.Disposable, (Color.FromRgb(43uy, 145uy, 175uy), Color.FromRgb(78uy, 220uy, 176uy)) ]
-
+        let colorData = // name,                      (light,                            dark)
+            [
+                FSharpClassificationTypes.MutableVar, (Color.FromRgb(160uy, 128uy, 0uy), Color.FromRgb(255uy, 210uy, 28uy))
+                FSharpClassificationTypes.Printf,     (Color.FromRgb(43uy, 145uy, 175uy), Color.FromRgb(78uy, 220uy, 176uy))
+                FSharpClassificationTypes.Disposable, (Colors.Tomato,                    Colors.Tomato)
```

Comment:
This won't be the actual color picked

Reply:
also lmao "picked" because it's a tomato

Reply:
aww why not keep tomato = disposable

Reply:
Yeah, I already love Tomato, `IDisposable` is scary, and tomatos are explosive, I like the combo :)